Makefile

```diff
@@ -94,8 +98,11 @@ endif
 ### rollout: rebuild and push docker image and restart cluster deployment
 rollout: docker restart
 
+### configure: configures already deployed controller according to set env variables
+configure: _update_yamls _apply_controller_cfg _reset_yamls
+
 ### update: update CRDs defined on cluster
-update: _update_yamls _update_crds _reset_yamls
+update: _update_crds
```

Comment:
remove updating yamls here since it does not modify CRDs
and sed output in misleading when `make update`

Reply:
Actually, looking at this more, we do need the `_update_yamls` step, since `_update_crds` reapplies the controller configmap, so this would reset any changes defined by env vars to their defaults.